src/Bridge/Standalone/Bootstrap.php

```diff
@@ -0,0 +1,466 @@
+     *   from given files.
+     * @param array $files
+     *   Configuration files, in override order in case of conflict.
+     *
+     * @return array
+     *   Merged proper configuration.
+     */
+    private static function configParse(array $config, array $files, LoggerInterface $logger): array
+    {
+        $configFileNames = ['db_tools.yaml', 'db_tools.yml', 'db_tools.config.yaml', 'db_tools.config.yml'];
+        $projectRoot = self::getProjectDir($logger);
+        $workdir = $config['workdir'] ?? $projectRoot ?? \getcwd();
+
+        // When no configuration file given, attempt to find one.
+        // @todo Should stop at first when found.
+        if (empty($files)) {
+            $candidates = [];
+            if ($projectRoot) {
+                foreach ($configFileNames as $filename) {
+                    $candidates[] = self::pathConcat($projectRoot, $filename);
+                }
+            }
+            if ($projectRoot !== $workdir) {
+                foreach ($configFileNames as $filename) {
+                    $candidates[] = self::pathConcat($workdir, $filename);
+                }
+            }
+            // Will not work under Windows (and that's OK).
+            if ($homedir = \getenv("HOME")) {
+                // @todo .config folder is configurable with XDG portals?
+                $candidates[] = self::pathConcat($homedir, '/.config/db_tools/config.yaml');
+                $candidates[] = self::pathConcat($homedir, '/.config/db_tools/config.yml');
+                // As dot files.
+                foreach ($configFileNames as $filename) {
+                    $candidates[] = self::pathConcat($homedir, '.' . $filename);
+                }
+            }
+
+            foreach ($candidates as $filename) {
+                if (\file_exists($filename)) {
+                    if (\is_readable($filename)) {
+                        $logger->notice("Found configuration file: {file}", ['file' => $filename]);
+                        $files[] = $filename;
+                    } else {
+                        $logger->warning("Configuration file could not be read: {file}", ['file' => $filename]);
+                    }
+                } else {
+                    $logger->debug("Configuration file does not exist: {file}", ['file' => $filename]);
+                }
+            }
+        }
+
+        $configs = [];
+        foreach ($files as $filename) {
+            $configs[] = self::configParseFile($filename);
+        }
+        $configs[] = $config;
+        $configs[] = self::configGetEnv();
+
+        // Use symfony/config and our bundle configuration, which allows us
+        // to use it fully for validation and merge.
+        $configuration = new StandaloneConfiguration();
+        $processor = new Processor();
+
+        $config = $processor->processConfiguration($configuration, $configs);
+
+        // Set a base directory for file and backup lookup.
+        $config['workdir'] ?? $workdir;
```

Comment:
What... stupid me.

Reply:
It is actually working because this line has nothing to do here: it would override the relative path resolution if the user don't specify a workdir. I simply removed it.